src/constructN.js

```diff
@@ -33,12 +31,22 @@ var nAry = require('./nAry');
  *      R.map(R.constructN(1, Widget), allConfigs); // a list of Widgets
  */
 module.exports = _curry2(function constructN(n, Fn) {
-    var f = function() {
-        var Temp = function() {}, inst, ret;
-        Temp.prototype = Fn.prototype;
-        inst = new Temp();
-        ret = Fn.apply(inst, arguments);
-        return Object(ret) === ret ? ret : inst;
-    };
-    return n > 1 ? curry(nAry(n, f)) : f;
```

Comment:
More to the point, it would cause an error, something I think your version will have to deal with as well:

```javascript
var Foo = function() {
    this.bar = 'baz';
};
var foo = R.construct(Foo);
var f = foo(); // TypeError: Function called with no arguments
```

The 1-case won't throw an error, but also doesn't do anything useful by currying, so why bother adding such a wrapper?